neevaco/neeva-android
# Force the app to enter Browser mode instead of going to the tab switcher

After https://github.com/neevaco/neeva-android/pull/1068/, we can get into a situation where the browser died in the background and sends the user to the tab switcher instead of the tab that is being created for the link in the external VIEW Intent.  Force the user to go to the browser in situations where it's obvious where there's a tab to switch to.

## Checklists

### How was this tested?

### Manual test cases
1. Kill the browser
2. Open an email in the gmail app that has a link in it
3. Click on the link
4. Watch the browser start up and load the page
5. See that the page shows the browser instead of the tab switcher

### Screenshots and screen recordings
Before:

https://user-images.githubusercontent.com/96092902/199090348-ebc5e3b6-23d1-4bf0-a464-6a69ac9d2df5.mp4

After:

https://user-images.githubusercontent.com/96092902/199090666-eb6d5249-6079-4205-9250-3f1b25745c57.mp4

## Issues addressed
* Fixes https://github.com/neevaco/neeva-android/issues/1086
